src/asymmetric_crypto_ctx.cpp

```diff
@@ -443,6 +461,7 @@ namespace {
         Key key;
     protected:
         ECDSASignatureFormat _sigFormat;
+        openssl::DigestTypes hashFunction;
```

Comment:
This library prefixes class variables with _

Reply:
Done